Zenodo.js

```diff
@@ -195,6 +195,11 @@ function scrape(doc, url) {
 
 			if (item.itemType == "document" && zoteroType[type]) {
 				item.itemType = zoteroType[type];
+				
```

Comment:
what would happen if we replace this whole thing, including l. 197 by `item.itemType = detectWeb(doc, url)`
That seems like a cleaner solution, provided our detect function is good.

Reply:
Could the `var zoteroType` block be removed as well, then?

Reply:
exactly, yes